package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clarity-js",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "description": "An analytics library that uses web page interactions to generate aggregated insights",
```

Comment:
I agree and it makes sense. Are you open to going back to 0.1.1 for the internal version tracking?